app/models/reservation.rb

```diff
@@ -141,6 +141,7 @@ def approved?
   def flagged?(flag)
     # checks to see if the given flag is set
     # you must pass the symbol for the flag
+    return unless FLAGS.key? flag
```

Comment:
Hm ok, I guess it only matters if we're checking for it on the sender's side but even if we're not there's really no harm in having that functionality built in.